canonical/canonical.com
# WD-12234 Created /data/opensearch/support

## Done

- Created /data/opensearch/support
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check if all the requirements are met for the new page.
- [Demo](https://canonical-com-1315.demos.haus/data/opensearch/support)
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=0-1&t=BSWoNixuloj9DSbO-0)
- [Copydoc](https://docs.google.com/document/d/1ovtXv8RBybw-flvttyqWOzZivUryQMP34T1UGe90HlY/edit)

## Issue / Card

Fixes # [WD-12234](https://warthogs.atlassian.net/browse/WD-12234)



[WD-12234]: https://warthogs.atlassian.net/browse/WD-12234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
